lockbox-ios/View/ItemDetailView.swift

```diff
@@ -91,6 +92,14 @@ extension ItemDetailView {
         leftButton.rx.tap
                 .bind(to: presenter.onCancel)
                 .disposed(by: self.disposeBag)
+
+        self.view.rx.swipeGesture(.right)
```

Comment:
1. Does this need a test to verify the expected behavior happens?
2. How possible is it to test such gestures?